docs/src/test-reporter-api/class-teststep.md

```diff
@@ -50,6 +50,16 @@ Start time of this particular test step.
 
 List of steps inside this step.
 
+## property: TestStep.attachments
```

Comment:
Seeing that every reporter implementation immediately turns this into indices, perhaps we should revisit the API and make this an array of numbers? This does not block the PR, but please put it as an explicit topic for the API review.